lib/components/Body.js

```diff
@@ -50,6 +56,15 @@ var Body = React.createClass({
     if (isDate(body)) {
       return $(Data, null, body.toString());
     }
+    if (isSet(body)) {
+      var contents = Array.from(body)
```

Comment:
Don't need to use a package for that. Besides, if we want to be ES5 only, we shouldn't use Function.name (as in your tests on `x.constructor.name`).